rotkehlchen/chain/ethereum/uniswap/typing.py

```diff
@@ -11,13 +15,21 @@
 from rotkehlchen.accounting.structures import Balance
 from rotkehlchen.assets.asset import EthereumToken
 from rotkehlchen.assets.unknown_asset import UnknownEthereumToken
+from rotkehlchen.chain.ethereum.trades import AMMTrade
 from rotkehlchen.constants import ZERO
 from rotkehlchen.fval import FVal
 from rotkehlchen.typing import (
     ChecksumEthAddress,
     Price,
 )
 
+log = logging.getLogger(__name__)
+
+SWAP_FEE = FVal('0.003')  # 0.3% fee for swapping tokens
```

Comment:
I can't think of a better way to do this. And so far this has not changed in uniswap. But if somehow UNI community decides to either increase or decrease the fees we will not notice it. 

Wish there was a way to pull the fee percentage from either the chain for a timestamp or from the graph.